java/core/src/java/org/apache/orc/impl/ColumnStatisticsImpl.java

```diff
@@ -2100,6 +2100,12 @@ public static ColumnStatisticsImpl deserialize(TypeDescription schema,
     return deserialize(schema, stats, true, true);
   }
 
+  public static ColumnStatisticsImpl deserialize(TypeDescription schema,
+                                                 OrcProto.ColumnStatistics stats,
+                                                 boolean writerUsedProlepticGregorian) {
+    return deserialize(schema, stats, writerUsedProlepticGregorian, true);
+  }
+
```

Comment:
If you don't mind, shall we remove this addition in order to keep the simplicity?

This new `static` function is used once and we can remove this new API like the following.
```java
- ColumnStatistics cs = ColumnStatisticsImpl.deserialize(null, statsProto, writerUsedProlepticGregorian);
+ ColumnStatistics cs = ColumnStatisticsImpl.deserialize(null, statsProto, writerUsedProlepticGregorian, true);
```

Reply:
Sure, makes sense. Just updated the PR